setup.py

```diff
@@ -0,0 +1,46 @@
+from setuptools import setup, find_packages
+import codecs
+import os
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+with codecs.open(os.path.join(here, "README.md"), encoding="utf-8") as fh:
+    long_description = "\n" + fh.read()
+
+VERSION = '{{VERSION_PLACEHOLDER}}'
+DESCRIPTION = "This package provides an API to query, filter and visualize disdrometer data."

```

Comment:
Suggestion: This package provides tools to homogenize, process, and analyze global disdrometer data.

Reply:
ok